src/executables/goloso-main.cpp

```diff
@@ -0,0 +1,26 @@
+//
+// Created by Christian nahuel Rivera on 11/11/19.
+//
+
+#include "../utils/util.hpp"
+#include "../entities/core/AdyacencyMatrixGraph.h"
+#include "../entities/entrypoint/InputCVRP.h"
+#include "../entities/entrypoint/OutputCVRP.h"
+#include "../utils/AlgorithmMain.h"
```

Comment:
Usar path relativo al incluir files:
```suggestion
#include "utils/util.hpp"
#include "entities/core/AdyacencyMatrixGraph.h"
#include "entities/entrypoint/InputCVRP.h"
#include "entities/entrypoint/OutputCVRP.h"
#include "utils/AlgorithmMain.h"
```